workflow-templates/im-test-k6-ci.yml

```
@@ -0,0 +1,24 @@
+name: K6 Automated Load Tester
+
+on: 
+  push:
+    branches: 
+      - master
+      - main
+
+env:
+  testFile: "tests/k6/check-health-simple.js"
```

@jsclifford Here is an example of the todo
```suggestion
  testFile: "" # TODO:  Add the name of the test file, like tests/k6/check-health-simple.js
```

---

Added recomendation.